### ISSUE TITLE
Orange should be used to indicate empty editor group focus in high contrast theme

### ISSUE THREAD

**Issue by Tyriar** [Microsoft/vscode#50710]
VS Code version: Code - Insiders 1.24.0-insider (41a7c4976ebd1f6ced20c034485300f46ab1829f, 2018-05-29T06:09:34.618Z)
OS version: Windows_NT x64 10.0.16299

---

#50458

Focus right now only lightens the logo and close button. I don't think this is dramatic enough for the HC theme:

![image](https://user-images.githubusercontent.com/2193314/40676173-cf32230a-632e-11e8-963f-79ba5cac0723.png)

**Comment by bpasero** [comment]
I ended up introducing a new theme color `editorGroup.activeEmptyBorder` which only in HC theme by default is defined as `focusBorder`.

**Comment by Tyriar** [comment]
@bpasero nice, this will solve the too subtle focus problem on my theme by matching the focus style of explorer (when no files are active):

![image](https://user-images.githubusercontent.com/2193314/40728289-a3ad7fb2-63de-11e8-8825-50f82b027d81.png)

**Comment by Tyriar** [comment]
Ah you're right, I actually want focus not active.

**Comment by bpasero** [comment]
I changed this color back to behave differently, only when the empty group has keyboard focus. Now its called: `editorGroup.focusedEmptyBorder`

**Comment by bpasero** [comment]
Verify with builds from #50458

**Comment by Tyriar** [comment]
@bpasero the orange is present in all themes, not just HC:

![image](https://user-images.githubusercontent.com/2193314/40795740-d1c8074e-64b7-11e8-865d-fd771322c3dc.png)

**Comment by Tyriar** [comment]
Verified, ignore the deleted comment/reopen. I defined it locally :smile:
